lib/index.js

```diff
@@ -39,7 +40,7 @@ async function runForGlobs(patterns, cwd, options = {}) {
   debug('componentPaths = %O', paths);
 
   let packageRoot = await pkgDir(cwd);
-  let pkgContent = fs.readFileSync(`${packageRoot}/package.json`);
+  let pkgContent = fs.readFileSync(fsPath.join(packageRoot, 'package.json'));
```

Comment:
AFAIK the `fs.readFileSync()` API takes care of this issue internally already. are you sure that this is needed?